src/_shared/components/CorpusItemCardImage/CorpusItemCardImage.tsx

```diff
@@ -118,6 +120,7 @@ const getBottomLeftLabel = (
           : curationPalette.overlayBgBlack,
       }}
       data-testid="last-scheduled-overlay"
+      onClick={toggleScheduleHistoryModal}
```

Comment:
I am a bit puzzled by this change - it seems that you've changed the component so that the both the Pocket pink overlays are now clickable further down in the code, but you're also passing the toggle function to the label helper function.

Reply:
No, I've moved the `onClick` to the inner box that only contains the 'last scheduled label'. The 'new domain' warning doesn't use a function to render it, so I've renamed the function to `getBottomLeftLastScheduledLabel` to be more specific about what it does.

Reply:
I'm happy to get suggestions about how to make the code more clear in our pairing time!

Reply:
Paired - decided to leave this as-is and make the new overlay not have the hand cursor when hovering over instead.